### PR TITLE
chore(demo-integrations): configure basic integration tests

### PR DESCRIPTION
**Pull request by vladimirpotekhin**
## PR Checklist

Please check if your PR fulfills the following requirements:

-   [ ] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0-beta.4/)
-   [ ] Tests for the changes have been added (for bug fixes / features)
-   [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

-   [ ] Bugfix
-   [ ] Feature
-   [ ] Refactoring (no functional changes, no api changes)
-   [ ] Other... Please describe:

## What is the current behavior?

Closes #

## Does this PR introduce a breaking change?

-   [ ] Yes
-   [ ] No

### CODE REVIEW

File: projects/demo-integrations/cypress/tests/demo-paths.ts [hunk 1]
```diff
@@ -0,0 +1,89 @@
+export const DEMO_PATHS = [
```
Comment (waterplea) [inline projects/demo-integrations/cypress/tests/demo-paths.ts:1]: Would be great if we could automate this stuff. Can we reach for routing config?

File: projects/demo-integrations/cypress/tests/shared.entities.ts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export const EXAMPLE_ID = 'tui-doc-example';
+export const PROPS_AND_METHODS =
+    '%D0%A1%D0%B2%D0%BE%D0%B9%D1%81%D1%82%D0%B2%D0%B0_%D0%B8_%D0%BC%D0%B5%D1%82%D0%BE%D0%B4%D1%8B';
```
Comment (waterplea) [inline projects/demo-integrations/cypress/tests/shared.entities.ts:3]: This looks like Russian tab name from proprietary portal. Should be `API`, I guess

File: projects/demo-integrations/cypress/tests/demo/exclusions.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
+const map = new Map([
+    ['components/select', [5, 6]],
```
Comment (waterplea) [inline projects/demo-integrations/cypress/tests/demo/exclusions.ts:2]: Why are we excluding those?

Comment (vladimirpotekhin) [reply projects/demo-integrations/cypress/tests/demo/exclusions.ts]: I remove the 6th example from the exclusion. 5th example makes the tests unstable due to the loader.

File: projects/demo-integrations/cypress/tests/shared.entities.ts (hunk 2 above)
Comment (vladimirpotekhin) [reply projects/demo-integrations/cypress/tests/shared.entities.ts]: Thanks, I delete this one. This is an unnecessary const for deep tests, I'll add it in a future PR

File: projects/demo-integrations/cypress/tests/demo-paths.ts (hunk 1 above)
Comment (vladimirpotekhin) [reply projects/demo-integrations/cypress/tests/demo-paths.ts]: Now the paths are taken from the constant `pages` from demo with exclusions

**Review by waterplea — APPROVED** [review]

**Pull request merged**